src/components/loginForm/LoginForm.js

```diff
@@ -0,0 +1,75 @@
+import React from "react";

```

Comment:
LoginForm only used by `pages/login/Login.js` so I propose to move this component to this location:

`pages/login/components/form/LoginForm.js`

Page-specific components should be placed inside of it's page folder.